From a688870e849fc64a2a81b089840b1469ca5d09ae Mon Sep 17 00:00:00 2001
From: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
Date: Sun, 24 Jan 2021 22:17:32 +0100
Subject: [PATCH 020/122] dt-bindings: arm: amlogic: add support for the WeTek
 Core

The WeTek Core is based on the Amlogic Meson8m2 (S812) SoC with the
following specifications:
- 2GB DDR3 RAM
- 8GB NAND flash
- Gbit/s Ethernet using an RTL8211F PHY
- AMPAK AP6330 802.11b/g/n + BT 4.0 SDIO wireless module
- 1 power button on the top of the case
- 1 reset button on the underside of the case
- HDMI 1.4
- Composite video output + 2-channel audio output using a 3.5mm jack
- Optical S/PDIF output
- 4x USB ports (two external, two internal) connected to the SoC through
  a GL850G USB hub
- remote receiver connected via USB on one of the internal USB ports
- IR receiver
- micro SD card slot
- UART pins (2.54mm headers need to be soldered) on the PCB

Signed-off-by: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
---
 Documentation/devicetree/bindings/arm/amlogic.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/amlogic.yaml b/Documentation/devicetree/bindings/arm/amlogic.yaml
index 6423377710ee..2a88b24384d0 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.yaml
+++ b/Documentation/devicetree/bindings/arm/amlogic.yaml
@@ -41,6 +41,7 @@ properties:
         items:
           - enum:
               - tronsmart,mxiii-plus
+              - wetek,core
           - const: amlogic,meson8m2
 
       - description: Boards with the Amlogic Meson8b SoC
-- 
2.25.1

